scripts/get-shapefiles.py

```diff
@@ -0,0 +1,204 @@
+# get-shapefiles.py, ircama, 2016-6-2
+
+import os
+import errno
+import tarfile,sys
+import zipfile
+import subprocess
+import distutils.spawn
+import argparse
+
+try:
+    import wget
+except ImportError:
+    sys.exit("""   Error: you need wget!
+   run 'pip install wget'
+   If you get an error running pip, try to run 'Scripts\pip install wget'
+   If you still get an error, run 'python -m pip install --upgrade pip' before.""")
+
+parser = argparse.ArgumentParser(description="This script generates and populates the 'data' directory with all needed shapefiles.")
+parser.add_argument('-q',"--quiet", dest='quiet_mode', action='store_true',
+                    help="don't print status messages to stdout")
+parser.add_argument('-u', "--update", dest='update_mode', action='store_true',
+                    help="allow updating an already existing 'data' directory")
+parser.add_argument('-n', "--nodownload", dest='nodownload', action='store_true',
+                    help="do not download files if already existing locally")
+parser.add_argument('-e', "--noextract", dest='noextract', action='store_true',
+                    help="do not populate target directories with downloaded data")
+parser.add_argument('-s', "--noshape", dest='noshape', action='store_true',
+                    help="do not run shapeindex")
+parser.add_argument('-1', dest='Owb',  action='store_true',

```

Comment:
variables like `--world-boundaries`, `--simplified-water`, `--ne-admin`, etc would be easier to remember than numerics